pywhat/Data/regex.json

```diff
@@ -151,6 +151,30 @@
          "Networking"
        ]
     },
+    {
+      "Name": "Amazon Web Services Access Key",
+      "Regex": "(?<![A-Z0-9])[A-Z0-9]{20}(?![A-Z0-9])",
+      "plural_name": false,
+      "Description": "Access key used to authenticate with the AWS API",
+      "Rarity": 1,
+      "URL": null,
+      "Tags": [
+         "Credentials",

```

Comment:
I just remembered!! We have an `AWS` tag, mind adding that? And if you spot any other AWS regex without it please add it :) AWS has so many (albeit, confusing) IDs that it needs its own tag haha!

Reply:
other than this, looks good to go 👍🏻

Reply:
Can do. Would you like me to add some of the other AWS types like security group, EC2 instance, organization ID? They'll be less common, but do have a fixed format.

Reply:
Sure, if you don't mind! :)

Reply:
Done... I'll see if I can think of others on my travels and will update as I go. Ok?